user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQIOnDestroy.md

```diff
@@ -1,11 +1,22 @@
 ---
-uid: IGQIOnDestroy
+uid: GQI_IGQIOnDestroy
 ---
 
-# IGQIOnDestroy
+# IGQIOnDestroy Interface
 
-The *IGQIOnDestroy* interface is called when the instance object is destroyed, which happens when the session is closed, e.g. in case of inactivity or when all the necessary data has been retrieved. It can for instance be used to end the connection with a database. This interface has one method:
+## Definition
+
+Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
+Assembly: `SLAnalyticsTypes.dll`
+
+The *IGQIOnDestroy* interface is called when the instance object is destroyed, which happens when the session is closed, e.g. in case of inactivity or when all the necessary data has been retrieved. It can for instance be used to end the connection with a database.
+
+> [!TIP]
+> See also: [IGQIOnInit](xref:GQI_IGQIOnInit)
+
+## Methods
+
+### OnDestroyOutputArgs OnDestroy(OnDestroyInputArgs args)
+
+This method is emitted when the data source or custom operator won't be used anymore. It can safely be destroyed.
```

Comment:
This line is a bit weird.
```suggestion
This method is invoked when the data source or custom operator instance won't be used anymore.
It can be implemented to clean up or dispose of any resources the instance may hold.
```